app/queries/site_query.rb

```diff
@@ -4,16 +4,16 @@ def order_by(params)
     key, direction = params[:sort]&.to_unsafe_h&.first
     direction = direction.to_s.downcase.to_sym.presence_in(directions) || directions.first
     case key
-    in :url
+    in "url"
       sortable_url = Arel.sql("REGEXP_REPLACE(audits.url, '^https?://(www\.)?', '')")
       subquery = model.with_current_audit
```

Comment:
Je me demande si on ne devrait pas utiliser `self` plutôt que `model`, pour conserver l'état de la query.
```suggestion
      subquery = self.with_current_audit
```

Reply:
Enfait ici on duplique avec le `self` puisque le `preload` est déjà call dans le controller. ... je verrais ça au refacto.